ceph/cluster.yaml

```diff
@@ -237,19 +237,19 @@ spec:
   healthCheck:
     daemonHealth:
       mon:
-        disabled: false
+        # disabled: false
         interval: 45s
       osd:
-        disabled: false
+        # disabled: false
         interval: 60s
       status:
-        disabled: false
+        # disabled: false
         interval: 60s
     # Change pod liveness probe, it works for all mon,mgr,osd daemons
     livenessProbe:
-      mon:
-        disabled: false
-      mgr:
-        disabled: false
-      osd:
-        disabled: false
+      mon: {}
+        # disabled: false
```

Comment:
@morningspace what is the default value for those parameters? Does the default value is still enabled?

Reply:
Yes, the default value is enabled. i.e.: `disabled: false`. In other words:

```
mon: {}
```

is identical to:

```
mon:
  disabled: false
```

It's been verified works.

@gyliu513